File: blog-gateway/src/main/java/at/technikumwien/MainApp.java
package at.technikumwien;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.zuul.EnableZuulProxy;

// see http://localhost:5555/actuator/routes

@SpringBootApplication
@EnableZuulProxy
public class MainApp {

    public static void main(String[] args) {
        SpringApplication.run(MainApp.class, args);
    }

}

File: bonus-service/src/main/kotlin/at/technikumwien/MainApp.kt
package at.technikumwien

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.stream.annotation.EnableBinding
import org.springframework.cloud.stream.messaging.Sink

@SpringBootApplication
@EnableBinding(Sink::class)
class BonusServiceApplication

fun main(args: Array<String>) {
    runApplication<BonusServiceApplication>(*args)
}
